scripts/mutest-format.py

```diff
@@ -0,0 +1,6 @@
+filename = "mutation_test_result.txt"
```

Comment:
If you need to skip `PASS` and `SKIP` lines you could simply do:

```bash
cat mutation_test_result.txt | grep -Ev "PASS | SKIP" | tee mutation_test_result.txt
```

Without the need of having this python script

Reply:
I see thank you!